package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dugite",
-  "version": "1.87.0",
+  "version": "1.88.0-0",
```

Comment:
Was there supposed to be a `-beta` included with this version?

Reply:
> `1.88.0-0` since that's the default behavior of npm version preminor

but i don't feel strongly. am happy to make it `-beta0` if you'd prefer

Reply:
> (used 1.88.0-0 since that's the default behavior of `npm version preminor`)
 
Aha, there's the missing context. I still kind of like having `beta` in here for extra visibility, and the way to do that doesn't feel too complex:

```
$ npm version preminor --preid=beta
```

Thoughts?

Reply:
sounds good! i'll update that and force push to get rid of that errant tag